.travis.yml

```diff
@@ -188,6 +184,14 @@ install:
 script:
   - travis_wait 45 suite/runsuite_wrapper.pl travis $EXTRA_ARGS
 
+after_script:
+  - sleep 10
+  - echo "--end--"
+
+after_failure:
+  - sleep 10
+  - echo "--end--"
```

Comment:
A comment explaining why these are here would be good.

Reply:
Done.